csu

```diff
@@ -13,411 +13,294 @@
+  echo "Building scratch images..."
   docker-compose exec nginx gulp scratch
 }
 
 # Collecting static files
-function collect_static {
-  printf "\nCollecting static files..."
+cmd_collect_static() {
+  echo
+  echo "Collecting static files."
   docker-compose exec django /docker_venv/bin/python3 ./manage.py collectstatic --no-input --clear
 }
 
 # Run shell
-function shell {
+dev_shell() {
   docker-compose exec django bash
 }
-function shell_help {
-  printf "   shell             Open shell to Django folder.\n"
-}
+defhelp -dev shell 'Open shell to Django folder.'
 
 # Run style checks
-function style {
-  printf "Running PEP8 style checker...\n"
+dev_style() {
+  echo "Running PEP8 style checker..."
   docker-compose exec django /docker_venv/bin/flake8
   pep8_status=$?
-  printf "\nRunning Python docstring checker...\n"
+  echo
+  echo "Running Python docstring checker..."
   docker-compose exec django /docker_venv/bin/pydocstyle --count --explain
   pydocstyle_status=$?
-  ! (( $pep8_status || $pydocstyle_status ))
-}
-function style_help {
-  printf "   style             Run style checks.\n"
+  ! (( pep8_status || pydocstyle_status ))
 }
+defhelp -dev style 'Run style checks.'
 
 # Run test suite
-function test_suite {
-  printf "Running test suite...\n"
+dev_test_suite() {
+  echo "Running test suite..."
   docker-compose exec django /docker_venv/bin/coverage run --rcfile=/cs-unplugged/.coveragerc ./manage.py test --settings=config.settings.testing --pattern "test_*.py" -v 3
 }
-function test_suite_help {
-  printf "   test              Run test suite with code coverage.\n"
-}
+defhelp -dev test 'Run test suite with code coverage.'
 
 # Run specific test suite
-function test_specific {
-  printf "Running specific test suite...\n"
+dev_test_specific() {
+  echo "Running specific test suite..."
   docker-compose exec django /docker_venv/bin/python3 ./manage.py test --settings=config.settings.testing "${1}" -v 3
 }
-function test_specific_help {
-  printf "   test_specific     Run specific test suite. Pass in parameter\n"
-  printf "                     of Python test module.\n"
-}
+defhelp -dev test_specific 'Run specific test suite. Pass in parameter of Python test module.'
 
 # Display test coverage table
-function test_coverage {
-  printf "Displaying test suite coverage...\n"
+dev_test_coverage() {
+  echo "Displaying test suite coverage..."
   docker-compose exec django /docker_venv/bin/coverage xml -i
   docker-compose exec django /docker_venv/bin/coverage report -m --skip-covered
 }
-function test_coverage_help {
-  printf "   test_coverage     Display code coverage report.\n"
-}
+defhelp -dev test_coverage 'Display code coverage report.'
 
 # Run test suite backwards for CI testing
-function test_suite_backwards {
-  printf "Running test suite backwards...\n"
-  if [ "$TRAVIS_PULL_REQUEST" != "false" ]
-  then
+dev_test_suite_backwards() {
+  echo "Running test suite backwards..."
+  if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
```

Comment:
You might encounter a merge conflict here. I added some stuff around this recently to make sure that when the command is not run under a pull request the result is 0, while if the command is run the result is that of the docker-compose command. The code might be superfluous, so please take time to consider it when merging it in.